tapeout/src/test/resources/macros/lib-2048x16-n28.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "sram",
+    "name": "vendor_sram_16",
```

Comment:
I copy and paste JSON files from pcad, only used for tests. Names are not important at all.